go.mod

```diff
@@ -67,7 +67,7 @@ require (
 	github.com/shopspring/decimal v1.3.1
 	github.com/smartcontractkit/caigo v0.0.0-20230621050857-b29a4ca8c704
 	github.com/smartcontractkit/chainlink-cosmos v0.4.1-0.20230831132059-42af68994512
-	github.com/smartcontractkit/chainlink-relay v0.1.7-0.20230912094934-cb83289b440e
+	github.com/smartcontractkit/chainlink-relay v0.1.7-0.20230912195355-fec1da7953fc
```

Comment:
I see that this commit is merged in for chainlink-relay but don't forget the title of this PR

Reply:
Er, nvm I just did to `fec1da7953fc340889554d22cc73ec2981f6eda9`